pkg/kubelet/dockertools/docker.go

```diff
@@ -316,7 +316,11 @@ func (d *dockerContainerCommandRunner) PortForward(podInfraContainerID string, p
 	}
 
 	containerPid := container.State.Pid
-	// TODO use exec.LookPath for socat / what if the host doesn't have it???
+	// TODO what if the host doesn't have it???
```

Comment:
I think the most appropriate way to handle this is to have the kubelet check for socat and return an error if it's not available when the port forwarding request comes in, before spawning listeners and so on. But that's not implemented now, and doesn't need to be part of this PR.